src/harrastuspassi/harrastuspassi/management/commands/import_lipas.py

```diff
@@ -36,18 +35,23 @@ class Command(BaseCommand):
     }
 
     def add_arguments(self, parser):
+
         parser.add_argument('--url', action='store', dest='url', default=settings.LIPAS_URL,
                             help='Import from a given URL')
+        parser.add_argument('--non_hp', action='store_false', dest='import_only_hp', default=True,
+                            help='Import also the places that have Harrastuspassi flag set to False')
 
     def handle(self, *args, **options):
         page_number = 1
         created_hobbies = 0
         updated_hobbies = 0
         created_locations = 0
         updated_locations = 0
-
+        
         self.stdout.write(f'Starting to pull data from {options["url"]}')
         start_time = time.time()
+        if options['import_only_hp']: 
+            options['url'] = f"{options['url']}?harrastuspassi=true"
```

Comment:
The query params are passed on line 77 as `params` kwarg. It is generally safer to let the request library to handle the construction of the URL.

`requests` may be flexible enough to survive even if it is passed query params both in the url and in params, but from review standpoint this looks scary and potentially has undefined behavior.

If we want to keep this `if` statement here, and not within the loop, we could do for example:

```python
extra_query_params = {}
if options['import_only_hp']:
    extra_query_params['harrastuspassi'] = 'true'
```

and in the loop:

```python
query_params.update(extra_query_params)
```

Reply:
Also, please point this PR to `develop` branch.